qiskit_nature/second_q/mappers/parity_mapper.py

```diff
@@ -51,4 +49,4 @@ def map(self, second_q_op: FermionicOp) -> PauliSumOp:
             b_x = np.asarray(b_x + [1] + [1] * (nmodes - i - 1), dtype=bool)
             pauli_table.append((Pauli((a_z, a_x)), Pauli((b_z, b_x))))
 
-        return QubitMapper.mode_based_mapping(second_q_op, pauli_table)
+        return FermionicMapper.mode_based_mapping(second_q_op, pauli_table)
```

Comment:
I guess I do not understand this change - the method is really part of the QubitMapper class (thats where its defined right) - while its possible to access it other ways I am wondering why since if you go to FermionicMapper you will not see the method

Reply:
Yes, this is a mistake, because I first removed the `FermionicMapper` class (used `QubitMapper` directly) and then re-introduced it. Will fix.